README.md

```diff
@@ -21,29 +34,53 @@ React Native Purchases is a client for the [RevenueCat](https://www.revenuecat.c
 💯 | Well maintained - [frequent releases](https://github.com/RevenueCat/purchases-ios/releases)  
 📮 | Great support - [Help Center](https://revenuecat.zendesk.com) 
 
+## Getting Started
+For more detailed information, you can view our complete documentation at [docs.revenuecat.com](https://docs.revenuecat.com/docs).
+
+Please follow the [Quickstart Guide](https://docs.revenuecat.com/docs/) for more information on how to install the SDK.
+
+Or view our React Native sample app:
+- [MagicWeather](examples/MagicWeather)
+
 ## Requirements
 
 The minimum React Native version this SDK requires is `0.64`.
 
+## SDK Reference
+Our full SDK reference [can be found here](https://revenuecat.github.io/purchases-ios-docs).
+
+## Contributing
+Contributions are always welcome! To learn how you can contribute, please see the [Contributing Guide](./Contributing/CONTRIBUTING.md).
+
+---
+
 ## Installation
 
 ExpoKit projects of version 33 or higher can successfully use react-native-purchases. If you haven't upgraded, you can follow [the instructions here to upgrade](https://docs.expo.io/versions/latest/expokit/expokit/#upgrading-expokit). 
 
-If you're planning on ejecting from Expo, upgrade your expo version _first_, THEN eject. It'll save you a whole lot of hassle.
+> ❗️ If you're planning on ejecting from Expo, upgrade your expo version _first_, THEN eject. It'll save you a whole lot of hassle. ❗️
 
-### Add the library to the project
+### 1. Add the library to the project
+
+```
+$ npm install react-native-purchases --save
+```
 
-`$ npm install react-native-purchases --save`
 or
-`$ yarn add react-native-purchases`
 
-### Link library to the project
+```
+$ yarn add react-native-purchases
+```
 
-`$ react-native link react-native-purchases`
+### 2. Link library to the project
+
+```
+$ react-native link react-native-purchases
+```
 
 ### Additional iOS Setup
 
-#### If your project uses Cocoapods
```

Comment:
This will make the header smaller than "Additional iOS Setup" which is what we want right?

Reply:
Interesting, the additional #'s were just not nesting any further on my preview, so I removed them.

I have reverted the headers.